### DISCUSSION TITLE
🚀 Feature request: Add dd.mm.yyyy date format

### DISCUSSION THREAD

**Discussion by Caleydon** [ExpressionEngine/ExpressionEngine#4540] [Feature Requests]
### Suggest an idea for ExpressionEngine

Hi. 

Can you please add **dd.mm.yyyy** date formatting as one of the options on the EE Settings page?

I found that 24 countries use this format.

Is there any issue with adding it? I asked for this date format a few years ago, and I received a response stating that it could cause problems.

After updating EE, I still need to add the option manually into Config.php in order to use this date formatting.
`'%d.%m.%Y' => 'dd.mm.yyyy',`

**Comment by intoeetive** [comment]
I don't see any potential issues with this, do you @robinsowell ?

PR: https://github.com/ExpressionEngine/ExpressionEngine/pull/4620

**Comment by Caleydon** [comment]
One comment: dd.mm.yyyy means that the date will be in the format, e.g., 01.01.2025 and not 1.1.2025. Am I right? Because implementation in the EE produce 1.1.2025 shape.

**Comment by intoeetive** [comment]
@Caleydon you are absolutely right. We have to fix that: https://github.com/ExpressionEngine/ExpressionEngine/pull/4670
